packages/components/src/components/hds/layout/flex/types.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+export enum HdsLayoutFlexDirectionValues {
+  Row = 'row',
+  Column = 'column',
+}
+
+export type HdsLayoutFlexDirections = `${HdsLayoutFlexDirectionValues}`;
+
+export enum HdsLayoutFlexJustifyValues {
+  Start = 'start',
+  Center = 'center',
+  End = 'end',
+  SpaceBetween = 'space-between',
+  SpaceAround = 'space-around',
+  SpaceEvenly = 'space-evenly',
+}
+
+export type HdsLayoutFlexJustifys = `${HdsLayoutFlexJustifyValues}`;
+
+export enum HdsLayoutFlexAlignValues {
+  Start = 'start',
+  Center = 'center',
+  End = 'end',
+  Stretch = 'stretch',
```

Comment:
```suggestion
  Stretch = 'stretch',
  Baseline = 'baseline',
```
[Question] Would it be useful to add `baseline` as an option? I don't see it being used in cloud ui at all, but looking at the audit there are ~20 instances of it in atlas.

Reply:
The `baseline` alignment is very tricky to understand, so better to keep things simple